libs/core/algorithms/include/hpx/parallel/datapar/find.hpp

```diff
@@ -67,22 +68,23 @@ namespace hpx { namespace parallel { inline namespace v1 { namespace detail {
     };
 
     template <typename ExPolicy, typename Iterator, typename Sentinel,
-        typename T, typename Proj = util::projection_identity>
-    inline constexpr std::enable_if_t<
-        hpx::is_vectorpack_execution_policy<ExPolicy>::value, Iterator>
-    tag_invoke(sequential_find_t<ExPolicy>, Iterator first, Sentinel last,
+        typename T, typename Proj = util::projection_identity,
+        HPX_CONCEPT_REQUIRES_(
+            hpx::is_vectorpack_execution_policy<ExPolicy>::value)>
+    HPX_HOST_DEVICE HPX_FORCEINLINE Iterator tag_invoke(
+        sequential_find_t<ExPolicy>, Iterator first, Sentinel last,
         T const& val, Proj proj = Proj())
     {
         return datapar_find<ExPolicy>::call(first, last, val, proj);
     }
 
     template <typename ExPolicy, typename FwdIter, typename Token, typename T,
-        typename Proj>
-    inline constexpr std::enable_if_t<
-        hpx::is_vectorpack_execution_policy<ExPolicy>::value, void>
-    tag_invoke(sequential_find_t<ExPolicy>, std::size_t base_idx,
-        FwdIter part_begin, std::size_t part_count, Token& tok, T const& val,
-        Proj&& proj)
+        typename Proj,
+        HPX_CONCEPT_REQUIRES_(
+            hpx::is_vectorpack_execution_policy<ExPolicy>::value)>
```

Comment:
We slowly modernize our code to C++17 wherever we can. Could you use `hpx::is_vectorpack_execution_policy_v<ExPolicy>` instead?

Reply:
I will do it in a separate PR at once for all the datapar algorithms. :)